build.go

```diff
@@ -272,6 +272,17 @@ func main() {
 
 	initTargets()
 
+	// we assume that getVersion should return valid version,
```

Comment:
Actually since you're here, let's not assume this, and actually verify we don't fuck up and build a broken version by mistake? :)

Reply:
OK, I guess that makes sense. Added new commit, which can safely be squashed upon merge to avoid unnecessary noise.